data-service/src/integrationTest/java/uk/gov/laa/ccms/data/IntegrationTestInterface.java

```diff
@@ -19,6 +19,11 @@ static void properties(DynamicPropertyRegistry registry) {
     registry.add("spring.datasource.url", oracleContainerSingleton.getOracleContainer()::getJdbcUrl);
     registry.add("spring.datasource.username", oracleContainerSingleton.getOracleContainer()::getUsername);
     registry.add("spring.datasource.password", oracleContainerSingleton.getOracleContainer()::getPassword);
+
+    registry.add("laa.ccms.springboot.starter.auth.authentication-header", () -> "Authorization");
```

Comment:
Should this not be in the integrationTest application.yml?

Reply:
Agree, that makes more sense. Moved.